oc_mirror_cleanup.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -x
```

Comment:
Would also prefer to see `set -euxo pipefail` here.

Reply:
Fixed